presto-raptor/src/main/java/com/facebook/presto/raptor/storage/organization/ShardRange.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor.storage.organization;
+
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+public class ShardRange
+{
+    private final Tuple minTuple;
+    private final Tuple maxTuple;
+
+    public static ShardRange of(Tuple min, Tuple max)
+    {
+        return new ShardRange(min, max);
+    }
+
+    public ShardRange(Tuple minTuple, Tuple maxTuple)
```

Comment:
Either make the constructor private or remove the factory (it doesn't seem useful for such a simple class)